jetty-start/src/main/java/org/eclipse/jetty/start/Main.java

```diff
@@ -325,12 +326,22 @@ public StartArgs processCommandLine(String[] cmdLine) throws Exception
         modules.registerAll();
 
         // 4) Active Module Resolution
-        for (String enabledModule : modules.getSortedNames(args.getEnabledModules()))
+        List<String> selectedModules = args.getEnabledModules();
+        List<String> sortedSelectedModules = modules.getSortedNames(selectedModules);
+        List<String> unknownModules = new ArrayList<>(selectedModules);
+        unknownModules.removeAll(sortedSelectedModules);
```

Comment:
The use of `modules.getSortedNames(List<String>)` actually left out unknown modules from the returned list.
This filtering to show unknown modules is how we now error out when an unknown module is selected to be enabled.